sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java

```diff
@@ -426,4 +551,11 @@ private static StackFrame getModelStackFrame(StackTraceElement stackTraceElement
         }
         return result;
     }
+
+    @VisibleForTesting
+    public static void clearInstance() {
```

Comment:
It's not about instance variables here, would need another name like clearStaticState or something.